internal/tui/tui.go

```diff
@@ -53,6 +57,24 @@ func Start() error {
 		}
 	}()
 
+	// Create a handler for terminal resize events
+	var lastWidth int
+	app.SetBeforeDrawFunc(func(screen tcell.Screen) bool {
+		width, _ := screen.Size()
+		currentScreenWidth = width
+
+		// If width crosses our breakpoint of 130, refresh the layout
+		if width != lastWidth {
+			RefreshChan <- struct{}{} // Trigger a refresh
+		}
```

Comment:
This condition could be better. It's being called every time, but if we
check width against specific breakpoints we can defer refresh

Reply:
This works, so I've just removed the confusing comment for now